hyperspy/tests/signal/test_eds_tem.py

```diff
@@ -135,7 +135,7 @@ def test_get_calibration_from(self):
 class Test_quantification:
 
     def setUp(self):
-        s = EDSTEMSpectrum(np.ones([2, 1024]))
+        s = EDSTEMSpectrum(np.ones([2, 2, 1024]))
         energy_axis = s.axes_manager.signal_axes[0]
```

Comment:
ok - this needs addressing in other ways then, there are two options:

1. If you know the width of the probe in a line scan presumably you can still define an area per position, so the method could be generalised to line scans?

2. If you really won't be able to use cross sections for line scans then the code needs to raise errors if you try to do so.

My gut instinct is that option 1 is preferable.